### DISCUSSION TITLE
SMTP Authentification does not work anymore - since OctoPrint 1.8

### DISCUSSION THREAD

**Discussion by Ingo-Knito** [berrystephenw/OctoText#124] [Q&A]
Since i have upgraded to the OctoPrint 1.8 and restored a backup, OctoText does not send mails anymore.
OctoText Version 0.3.3 is running. In the past OctoText worked with perfect the gmx.net-server.

Now i have tried two server (smtp-mail.outlook.com and mail.gmx.net), but both of them run in the same error "Exception while talking to your mail server, check your SMTP settings.".

Any ideas?

**Comment by Bendermk6** [comment]
Just try the new 0.3.4rc, Steve had a error in the 0.3.3.

Cheers
Adam

**Comment by Ingo-Knito** [comment]
Thanks! Now is it working again :-)

Great work, this plugin!

**Comment by peterlanoie** [comment]
I have been getting failures for several months. Things were working fine then they just stopped. 
I am trying to send through my Gmail account using the Gmail SMTP configuration as described by all the "how to" articles I've found, but it keep failing. 

Current setup:

- OctoPrint 1.8.0 Python 3.7.3 OctoPi 0.18.0
- OctoText (0.3.3.1) (Just upgraded)

I have regenerated a new Gmail app password to ensure it wasn't just that.
I have also tried using my full Gmail address as the auth username with a blank host name.

The OctoPrint UI error message is ambiguous (just "Check your login and password") but I believe the actual authentication is OK.
The detailed log suggests this is a problem with Gmail specifically due to denial of mail relay:

```
Exception while logging into SMTP server(send_email_message) (550, b"5.7.0 Mail relay denied [***.***.***.***]. 
Invalid credentials for relay for\n5.7.0 one of the domains in: [127.0.1.1], 0.1.1], 1.1] (as obtained from\n5.7.0 
HELO and MAIL FROM).\n5.7.0 Email is being sent from a domain or IP address which isn't registered\n5.7.0 
in your G Suite account. Please login to your G Suite account and\n5.7.0 verify that your sending device IP 
address has been registered within\n5.7.0 the G Suite SMTP Relay Settings. For more information, please 
visit\n5.7.0  https://support.google.com/a/answer/6140680#maildenied p21-20020a02b395000000b0032e8480d25csm230229jan.35 
- gsmtp", '**********@gmail.com')
```

Unfortunately, all the help and documentation I'm finding for this problem is focused on G Suite settings which of course I don't have as a freemail user.

For the sake of testing, I also tried to send just to my own Gmail account (instead of my SMS address) to see if it would accept it. No luck there either.

**Reply by berrystephenw** [reply]
Wow, a whole nother level of spam protection. I don't know if you read this description: https://support.google.com/mail/answer/10336?visit_id=637885713482464448-1616441348&p=NotAuthorizedError&rd=1

from that it sounds like your ISP might be blocking sending email. Maybe try contacting them?

**Reply by peterlanoie** [reply]
I had not seen that article yet. Thanks for the additional info. I'll have to do some digging.
Sadly, I'll likely not try to contact my ISP. I don't like them, and they don't seem to like their customers much.
But I'll see if I can utilize an ISP relay for this. Otherwise, I might have to just the "blow and air horn when your print is done" plugin.
